src/Strategy/Erlang/Rebar3Tree.hs

```diff
@@ -25,10 +25,10 @@ import Types
 discover :: HasDiscover sig m => Path Abs Dir -> m ()
 discover = walk $ \dir _ files -> do
   case find (\f -> (fileName f) == "rebar.config") files of
-    Nothing -> pure ()
-    Just _  -> runSimpleStrategy "erlang-rebar3tree" ErlangGroup $ analyze dir
-
-  pure WalkContinue
+    Nothing -> pure WalkContinue
+    Just _  -> do
+      runSimpleStrategy "erlang-rebar3tree" ErlangGroup $ analyze dir
+      pure WalkSkipAll
```

Comment:
This is the fix for the rebar3tree issue

Reply:
Why does this fix the issue? Shouldn't we continue walking to find other `rebar.config` files?

Reply:
Specifically, the issue is that dependencies are downloaded into a subdirectory, and those dependencies _also_ contain `rebar.config`s, e.g., (`.rebar/deps` isn't accurate here; I forget what the directories are actually called)
```
myproject/rebar.config
myproject/.rebar/deps/packageA/rebar.config
myproject/.rebar/deps/packageB/rebar.config
```

..so we end up treating those as direct dependencies.

Worse though, anytime `rebar3 tree` is called on one of those dependencies, it then downloads _another level_ of dependencies under it, which also contain `rebar.config`s:

```
myproject/rebar.config
myproject/.rebar/deps/packageA/rebar.config
myproject/.rebar/deps/packageA/.rebar/deps/packageC/rebar.config
myproject/.rebar/deps/packageB/rebar.config
myproject/.rebar/deps/packageB/.rebar/deps/packageD/rebar.config
```

and this expands an additional level each time we run analysis

---

I don't know if the correct option is to ignore `rebar.config` in subdirectories, or just to blacklist the `.rebar/deps` (or whatever it's actually called)

Reply:
Ah, this makes sense. I like your change for now and we can revert or modify in the future. Thanks for the very detailed response.